vraptor-core/src/test/java/br/com/caelum/vraptor/environment/DefaultEnvironmentTest.java

```diff
@@ -137,11 +138,26 @@ public void shouldUseSystemPropertiesWhenSysenvIsntPresent() {
 	}
 	
 	public void shouldGetOverridedSystemPropertyValueIfIsSet() throws IOException {
-		DefaultEnvironment defaultEnvironment = new DefaultEnvironment(EnvironmentType.DEVELOPMENT);
+		DefaultEnvironment defaultEnvironment = buildEnvironment(EnvironmentType.DEVELOPMENT);
 		System.setProperty("env_name", "customEnv");
 		String value = defaultEnvironment.get("env_name");
 		assertThat("customEnv", is(value));
 		//unset property to not break other tests
 		System.setProperty("env_name", "");
 	}
+	
+	private DefaultEnvironment buildEnvironment(EnvironmentType environmentType) {
+		DefaultEnvironment defaultEnvironment = new DefaultEnvironment(environmentType);
+		defaultEnvironment.setup();
```

Comment:
you don't need to do that if you're already calling `loadProperties` in the public constructor, right? https://github.com/caelum/vraptor4/pull/973/files#diff-b7b62c45c7478eb642acfb81c12a4227R82